### PR TITLE
Travis: split aom and dav1d decode tests

### PR DESCRIPTION
**Pull request by barrbrain**
- [x] ~~For now choose `decode_test_dav1d`, it should be faster.~~
- [ ] ~~Ensure that the skipped variant can be built.~~
- [x] Just split the ignored tests

### CODE REVIEW

**Comment by shssoichiro** [comment]
I think we need a discussion on this. I was asked specifically to run both, I believe by @YaLTeR (although I'm bad at remembering who tells me things).

**Comment by barrbrain** [comment]
The purpose of the first patch is to gauge the impact of running both. If it is significant, then we may need to find the balance between CI responsiveness and avoiding silent breakage.

**Comment by YaLTeR** [comment]
Another thing we could try is run aom and dav1d ignored tests in separate jobs. This way they should run in parallel, potentially giving a speed-up?

**Comment by shssoichiro** [comment]
There's a little bit of waste from having to build rav1e twice. But hopefully not too bad, especially if the cache is used. That would be faster (based on wall time).

**Comment by lu-zero** [comment]
Appveyor had a timeout.

File: .travis.yml [hunk 1]
```diff
@@ -68,7 +68,7 @@ jobs:
         - which cargo-kcov || cargo install cargo-kcov
         # We need to run cargo clean to avoid caching issues
         - cargo clean
-        - RUSTFLAGS="-C link-dead-code" cargo build --features=decode_test,decode_test_dav1d,quick_test --tests --verbose
+        - RUSTFLAGS="-C link-dead-code" cargo build --features=decode_test_dav1d,quick_test --tests --verbose
```
Comment (shssoichiro) [inline .travis.yml]: Do we want to keep both enabled for the quick tests?

Comment (barrbrain) [reply .travis.yml]: Yes, I think I'll reduce this down to just splitting the ignored tests job.

**Pull request merged**